components/AddTask.js

```diff
@@ -1,14 +1,40 @@
+import { useState } from 'react'
+import axios from '../utils/axios'
+import { useAuth } from '../context/auth'
+import {toast} from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+
+toast.configure()
 export default function AddTask() {
-  const addTask = () => {
-    /**
-     * @todo Complete this function.
-     * @todo 1. Send the request to add the task to the backend server.
-     * @todo 2. Add the task in the dom.
-     */
+  const [Todotxt, setTodotxt] = useState("");
+  const { config,API_BASE_URL} = useAuth();
+  const addTask = (e) => {
+    e.preventDefault()
+    if (!Todotxt || Todotxt=="") {
+        return;
```

Comment:
Should have added some Error or Warning Toast

Reply:
Added error message in add task. Please, review again.